_posts/07-04-01-Interacting-via-Code.md

```diff
@@ -0,0 +1,101 @@
+---
+isChild: true
+title: Interakcija sa bazom podataka
+anchor: databases_interacting
+---
+
+## Interakcija sa bazom podataka {#databases_interacting_title}
+
+Kada programeri počnu da uče PHP, obično dođu u situaciju da mešaju logiku interakcije sa bazom i
+prezentacione logike, kao na primer u ovom slučaju:
+
+{% highlight php %}
+<ul>
+<?php
+foreach ($db->query('SELECT * FROM table') as $row) {
+    echo "<li>".$row['field1']." - ".$row['field1']."</li>";
+}
+?>
+</ul>
+{% endhighlight %}
+
+Ovo je loša praksa iz više razloga, a ponajviše zbog toga što je ovakav kôd teško debug-ovati,
+testirati i uopšte čitati.
```

Comment:
Nedostaje _it is going to output a lot of fields if you don't put a limit on there._